### DISCUSSION TITLE
Lots of empty space with only few entries

### DISCUSSION THREAD

**Discussion by weisJ** [Anze/KeyCluCask#15] [Ideas]
If there aren't a lot of entries present the window looks visually unbalanced with all the empty space in the lower half:

![IMG_7178](https://github.com/Anze/KeyCluCask/assets/31143295/703acc9f-a7a6-4772-b574-bcf6a5b21331)

I think the amount of columns and window width/height should be dynamic based on the amount of content to avoid empty space.

**Comment by Anze** [comment]
At the moment, the current behaviour is intentional, as it was inspired by other app that share the same approach. At first sight it was also not that impressive, but I quickly get used to it.
Still, If more people express the need for this improvement, I'll be happy to reevaluate and explore the possibility of implementing it in the future.

**Comment by mertandmarcus** [comment]
I agree. Regardless of what other apps were doing, this is what will take the app one step further. There's no reason blocking 60% of the screen only for 10 entries. KeyClu's own panel looks so neat.

**Comment by nickali** [comment]
I'm not against the shortcuts panel taking up all that real estate, but it would be nice if the width of persistent panel was greatly reduced.

**Comment by Anze** [comment]
Hey guys,
I've recently worked on making the panel's width and height adjustable. It doesn't autoresize, still it does offer manual control over the panel size.
